src/Native/Runtime/amd64/InteropThunksHelpers.S

```diff
@@ -10,22 +10,22 @@
 LEAF_ENTRY RhpCommonStub, _TEXT
 
     PUSH_ARGUMENT_REGISTERS
+    push_register    r10
 
-    // +8 for alignment
-    alloc_stack    (SIZEOF_MAX_FP_ARG_SPILL + 8)
```

Comment:
I would also prefer using SIZEOF_FP_REGS name, I didn't know we have constant like that already.